p5js/FaceApi/FaceApi_Image_Landmarks/sketch.js

```diff
@@ -0,0 +1,116 @@
+let faceapi;
+let img;
+let detections;
+
+// by default all options are set to true
+const detection_options = {
+    withFaceLandmarks: true,
+    withFaceExpressions: false,
+    withFaceDescriptors: false,
+}
+
+function preload(){
+    img = loadImage('assets/frida.jpg')
+}
+
+function setup() {
+    createCanvas(200, 200);
+    img.resize(width, height);
+
+    faceapi = ml5.faceApi(detection_options, modelReady)
+    textAlign(RIGHT);
+}
+
+function modelReady() {
+    console.log('ready!')
+    console.log(faceapi)
+    faceapi.detectSingle(img, gotResults)
+
+}
+
+function gotResults(err, result) {
+    if (err) {
+        console.log(err)
+        return
+    }
+    // console.log(result)
+    detections = result;
+
+    // background(220);
+    background(255);
+    image(img, 0,0, width, height)
+    if (detections) {
+        // console.log(detections)
+        drawBox(detections)
+        drawLandmarks(detections)
+    }
+}
+
+function drawBox(detections){
+    const {alignedRect} = detections;
```

Comment:
The destructuring syntax will probably be unfamiliar to people coming from intro p5.

Reply:
Right! I can change these to the variable and dot syntax